conf/ocsci/ceph_upgrade_params.yaml

```diff
@@ -0,0 +1,5 @@
+ENV_DATA:
+  skip_upgrade_checks: true
```

Comment:
Please document it and move some comment at the start of the file for what it's good for - this config file - you can point also to ticket.

Reply:
done